colorbleed/plugins/maya/publish/validate_maya_units.py

```diff
@@ -44,7 +48,5 @@ def repair(cls):
         current_linear = cmds.currentUnit(query=True, linear=True)
         cls.log.debug(current_linear)
 
-        cls.log.info("Setting time unit to 'PAL'")
-        cmds.currentUnit(time="pal")
-        current_time = cmds.currentUnit(query=True, time=True)
-        cls.log.debug(current_time)
+        cls.log.info("Setting time unit to match project")
+        mayalib.set_project_fps()
```

Comment:
How about letting the maya lib expose a more generic method `set_scene_fps(fps)` and you just pass the correct fps to the method? Would simplify it and make it more explicit on all ends?

Reply:
Yes! Let's do it.